what's new.md

```diff
@@ -205,6 +205,7 @@
 - Sometimes duplicate controller models are loaded in VR ([TrevorDev](https://github.com/TrevorDev))
 - Particle emit rate and start size over time do not reset on every particle system start ([TrevorDev](https://github.com/TrevorDev))
 - SSAO2 now correctly takes diffuse texture alpha value into account ([CraigFeldspar](https://github.com/CraigFeldspar))
+- Sandbox will now load assets relatively path-ed to same folder([Kyle Belfort](https://github.com/belfortk))

```

Comment:
It's not obvious. You want to update the one in `dist/preview release` instead of this one. I'm not sure why we have a `what's new.md` at the root.

Reply:
My bad. I missed that in Nick's comment.

Reply:
Updated!